npm/react/cypress/component/basic/error-boundary-spec.js

```diff
@@ -1,50 +1,50 @@
-/// <reference types="cypress" />

```

Comment:
Change from running `git add --renormalize . ` after adding .gitattributes file to normalize line endings